src/main/java/net/pms/encoders/FFmpegAudio.java

```diff
@@ -201,8 +201,9 @@ public ProcessWrapper launchTranscode(
 		}
 
 		// Streaming support
-		cmdList.add("-movflags");
-		cmdList.add("+faststart");
+//		cmdList.add("-movflags");
+//		cmdList.add("frag_keyframe+empty_moov");
+		
```

Comment:
@Nadahar The worst, is that i didn't used Google translate 8-/
It's look like it's clear only for me...so thanks for pointing that out, but that lead me with english lesson choice or maybe that Google Translate will be better in my case ;-)